govtool/frontend/src/components/organisms/GovernanceActionDetailsCardData.tsx

```diff
@@ -90,7 +91,21 @@ export const GovernanceActionDetailsCardData = ({
   const { screenWidth } = useScreenDimension();
   const { isMobile } = useScreenDimension();
 
-  const nonNullProtocolParams = replaceNullValues(epochParams, protocolParams);
+  const updatableProtocolParams = useMemo(
+    () =>
+      filterUpdatableProtocolParams(epochParams, protocolParams, [
+        "id",
+        "registered_tx_id",
+        "key",
```

Comment:
```suggestion
        "key",
        "nonce",
```
This was also present https://sanchogov.tools/governance_actions/84fe1da5d14450537db36c28ffe2a1a4878f501fc26c998bba54e396711205bc#0

Reply:
ah wait, is this the wrong place?

Reply:
Yep, I am filtering them from the right, not from the left. Not sure if that is the best approach to be honest. And in the params we could also update costModels if I'm correct (but they are not present in the data provided from db-sync).

For now one I am comparing the incoming protocolParams with the epoch params as after investigation they match to each other

Reply:
With some exceptions in which 3rd argument of that util helps

Reply:
understood!
thanks